### PR TITLE
Import processing from commons

### PR DESCRIPTION
**Pull request by Siegrift**
Closes https://github.com/api3dao/airnode/issues/1896

### CODE REVIEW

File: packages/airnode-node/src/api/index.ts [hunk 1]
```diff
@@ -295,9 +297,14 @@ export async function callApi(payload: ApiCallPayload): Promise<LogsData<ApiCall
...
   if (verificationResult) return verificationResult;
 
-  const processedPayload = await preProcessApiSpecifications(payload);
-  const ois = payload.config.ois.find((o: OIS) => o.title === payload.aggregatedApiCall.oisTitle)!;
-  const endpoint = ois.endpoints.find((e: Endpoint) => e.name === payload.aggregatedApiCall.endpointName)!;
+  const {
+    aggregatedApiCall: { parameters },
+  } = payload;
+  const ois = payload.config.ois.find((o) => o.title === payload.aggregatedApiCall.oisTitle)!;
+  const endpoint = ois.endpoints.find((e) => e.name === payload.aggregatedApiCall.endpointName)!;
```
Comment (dcroote) [inline packages/airnode-node/src/api/index.ts:304]: Why were the types (and their corresponding imports) removed from this line and the one above it?

File: packages/airnode-node/src/api/index.ts [hunk 2]
```diff
@@ -240,7 +242,7 @@ export async function processSuccessfulApiCall(
   if (!goExtractAndEncodeResponse.success) {
     const log = logger.pend('ERROR', goExtractAndEncodeResponse.error.message);
     // The HTTP gateway is a special case for ChainAPI where we return data from a successful API call that failed processing
-    if (payload.type === 'http-gateway') {
+    if (type === 'http-gateway') {
```
Comment (dcroote) [inline packages/airnode-node/src/api/index.ts:245]: Nice find

**Review by dcroote — APPROVED** [review]
Looks good; one minor question

File: packages/airnode-node/src/api/index.ts (hunk 1 above)
Comment (Siegrift) [reply packages/airnode-node/src/api/index.ts:304]: They were redundant. OIS is typed so there was no need to have them.

**Pull request merged**